src/test/java/pl/byczazagroda/trackexpensesappbackend/WalletTestIT.java

```diff
@@ -0,0 +1,32 @@
+package pl.byczazagroda.trackexpensesappbackend;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.MediaType;
+import org.springframework.test.web.servlet.request.MockMvcRequestBuilders;
+import pl.byczazagroda.trackexpensesappbackend.dto.CreateWalletDTO;
+import pl.byczazagroda.trackexpensesappbackend.repository.WalletRepository;
+
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+public class WalletTestIT extends BaseControllerTestIT {
+
+    @Autowired
+    private WalletRepository walletRepository;
+
+    @Test
+    @DisplayName("It should create wallet")
+    void shouldCreateWallet() throws Exception {
```

Comment:
you have to be sure the table is empty before start test

Reply:
Okay, done